internal/config/config.go

```diff
@@ -58,12 +65,16 @@ var Features = &config.FeatureFlags
 func init() {
 	var err error
 
+	viper.SetDefault("app.name", "provisioning")
+	viper.SetDefault("app.pathPrefix", "api")
```

Comment:
I find this a bit unreadable, would you mind refactoring this to just app.prefix being simply "/api/provisioning"? Ideally this value would have a validator that it is actually an absolute path.

Reply:
well app name is configured through Clowder, but the prefix (api) is "fixed" so I'd rather go with having static prefix and configure only the name if that would look better